nifi-manifest/nifi-extension-manifest-parser/pom.xml

```diff
@@ -29,11 +29,23 @@
             <artifactId>nifi-extension-manifest-model</artifactId>
             <version>1.16.0-SNAPSHOT</version>
         </dependency>
-        <dependency>
-            <groupId>com.fasterxml.jackson.dataformat</groupId>
-            <artifactId>jackson-dataformat-xml</artifactId>
```

Comment:
Is there a particular reason for switching from Jackson XML to JAXB?

Reply:
Yes, this module needs to be used from the framework and jackson-dataformat-xml brings in transitive dependency for woodstox which then gets found as an XML provider for other things that it wasn't intended to be found for. Using JAX-B seemed like a simpler approach since everything is already setup for it being available on the classpath.

Reply:
Thanks for the background, swithcing to JAX-B sounds like the best approach given the other dependencies you mentioned.